ui/nuxeo-document-permissions/nuxeo-document-permissions.js

```diff
@@ -127,14 +128,14 @@ import './nuxeo-popup-permission.js';
               <div class="actions">
                 <dom-if if="[[!_empty(inheritedAces)]]">
                   <template>
-                    <paper-button id="block" on-click="blockInheritance">
+                    <paper-button id="block" on-click="blockInheritance" class="primary small">
```

Comment:
Is it supposed to be `small` here?

Reply:
yes, for the permissions tab we discussed with @carolinagouveia to have the btns small. Not sure if there's a specific guideline, I think we're handling it case-by-case.